python/rmm/_lib/lib.pxd

```diff
@@ -65,10 +66,11 @@ cdef extern from "rmm/rmm.h" nogil:
         PoolAllocation = 1,
         CudaManagedMemory = 2,
 
-    ctypedef struct rmmOptions_t:
+    cdef cppclass rmmOptions_t:
```

Comment:
Needs to be declared a `cppclass` so that members are initialized with default constructors